wiki_articles/vscode.md

```diff
@@ -0,0 +1,33 @@
+# Getting Started With VS Code for C/C++
+
+## DISCLAIMER FOR BEGINNERS
+Do not use VS Code.
+It's better to use an IDE that works out of the box:
+- [Visual Studio Community](https://visualstudio.microsoft.com/vs/community/) for Windows (free)
+- [CLion](https://www.jetbrains.com/clion/) on any platform (free for students)
+- [XCode](https://developer.apple.com/xcode/) on macOS (free)
+- [repl.it](https://replit.com/site/ide) online IDE (free for public projects)
+
+## 1. Install The C/C++ Extension by Microsoft
+Install [the extension](https://code.visualstudio.com/docs/languages/cpp) and
+follow the tutorial.
+Don't use third-party plugins like *Code Runner*, use the official stuff.
+
+## 2. Install A Compiler (possibly GCC through MinGW, or GCC through WSL)
+You still need to install a compiler, which is the thing that turns your code
+into an executable, separately.
+Consider using
+[Windows Subsystem for Linux](https://code.visualstudio.com/docs/cpp/config-wsl)
+instead of MinGW, since Linux makes development much easier.
```

Comment:
If you want to include steps for setting up gcc, it's fairly straightforward
```
Install msys2 https://www.msys2.org/
Add C:\msys64\usr\bin and C:\msys64\ucrt64\bin to the PATH
Open a new powershell window (to get PATH changes)
pacman -S mingw-w64-ucrt-x86_64-gcc make cmake gdb rsync
g++ --version
```

Reply:
It's too much for an embed; the shorter the better. I also don't want to make this guide too Windows-specific; those lines are of little use to Mac or Linux users.